apps/modernization-ui/src/apps/page-builder/components/DeleteQuestion/DeleteQuestion.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useRef } from 'react';
+import { ButtonGroup, ModalFooter, ModalRef, Icon, ModalToggleButton } from '@trussworks/react-uswds';
+import { ModalComponent } from 'components/ModalComponent/ModalComponent';
+import './DeleteQuestion.scss';
+import { AlertBase } from '../AlertBase/AlertBase';
+
+type CommonProps = {
+    onDelete?: () => void;
+};
+
+const DeleteQuestion = ({ onDelete }: CommonProps) => {
+    const deleteModalRef = useRef<ModalRef>(null);
+
+    const handleDeleteQuetions = () => {
+        onDelete && onDelete();
+    };
+    return (
+        <>
+            <ModalToggleButton modalRef={deleteModalRef} className="delete-btn" type="submit" unstyled>
+                <Icon.Delete style={{ cursor: 'pointer' }} className="primary-color" />
+            </ModalToggleButton>
+            <ModalComponent
+                modalRef={deleteModalRef}
+                isLarge
+                modalHeading={'Confirmation'}
+                modalBody={
+                    <>
+                        <div style={{ padding: '0 24px' }}>
+                            <AlertBase
+                                type="warning"
+                                header="Are you sure you want to delete the question?"
+                                message="Deleting this question cannot be undone. Are you sure you want to continue?"
+                            />
```

Comment:
Issue(blocking): Can use ConfirmationModal since it is there, there will be another story to fix the design so it is consistent with the UX